src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -45,7 +47,31 @@ export type SpeciesProjectsResult = {
 };
 
 type SpeciesProjectsSearchResult = {
```

Comment:
Since we change this type, we need to update the table columns and the add/remove projects to use this model

Reply:
This is the type returned by the server, the hydrated type `SpeciesWithParticipantProjectsSearchResponse` below is what is used in the tables